### DISCUSSION TITLE
Charge schemes

### DISCUSSION THREAD

**Discussion by icamps** [grimme-lab/xtb#680] [Q&A] [answered]
Hello,

I would like to know which are the charge scheme used by xtb.

If running with GFN1-xTB, we got the Mulliken and CM5 charges but when running with GFN2-xTB, I just got the charges, without any other information about the scheme used.

Regards,

Camps

**Comment by haneug** [comment, marked as answer]
These charges should be Mulliken charges which are briefly mentioned in the [GFN2-xTB paper](https://pubs.acs.org/doi/10.1021/acs.jctc.8b01176).

**Reply by haneug** [reply]
You can also find this in the scc routine:
https://github.com/grimme-lab/xtb/blob/1922dd31cc734edb136d3ef5ea42c7d542170e8d/src/scc_core.f90#L484-L489
and 
https://github.com/grimme-lab/xtb/blob/1922dd31cc734edb136d3ef5ea42c7d542170e8d/src/scc_core.f90#L1304-L1326

**Reply by icamps** [reply]
The Mulliken charges are used even when I add `mulliken=false` in my input file?

**Reply by haneug** [reply]
The Mulliken charges are a core component of GFN2-xTB and can not be turned off to my knowledge.
